flipperdevices/flipperzero-firmware
# [FL-3374] iButton: delete key text overflow

# What's new

- Use widget_add_text_box_element above widget_add_string_element
- Correct height for title and uid

# Verification 

- Saved iButton key with long key
- Go to Delete scene (like iButton -> saved -> Choose Key -> Delete)
- See what long text not overflow

# Checklist (For Reviewer)
